bench/cli.mjs

```
@@ -0,0 +1,152 @@
+  process.exit(1);
+}
+
+// ── Run benchmarks ─────────────────────────────────────────────────────────
+
+console.log(`\nRunning ${filtered.length} benchmark(s) with size=${size}...\n`);
+
+const bench = new Bench({ time: 200 });
+
+for (const scenario of filtered) {
+  const label = `${scenario.name} ${scenario.operation}`;
+  const beforeHeap = process.memoryUsage().heapUsed;
```

ESLint

[error] 'process' is not defined.

(no-undef)